tutorial_plugin_python.md

```diff
@@ -0,0 +1,157 @@
+﻿---
```

Comment:
Most from above applies here as well. Will only check after the changes above are also integrated here.

Reply:
Looks mostly good, same as above. But also formatting issues.